File: ArkhamAsylum.Lib/Models/Patient.cs
using ArkhamAsylum.Lib.Services.CountryConnectors;
using Newtonsoft.Json;
using System.Collections.Generic;

namespace ArkhamAsylum.Lib.Models
{
    public class Patient : Entity
    {
        public string Name { get; set; }
        public string FirstSurname { get; set; }
        public string SecondSutname { get; set; }
        [JsonIgnore]
        public ICollection<Record> Records { get; set; }
    }
}

File: ArkhamAsylum.Web/wwwroot/Scripts/Models/Patient.js
class Patient extends Entity
{
    constructor(json)
    {
        super(json)
        if (json)
        {
            this.Name = json.name;
            this.FirstSurname = json.firsSurname;
            this.SecondSurname = json.secondSurname;
        }
        else
        {
            this.Name = "";
            this.FirstSurname = "";
            this.SecondSurname = "";
        }
    }
}